### DISCUSSION TITLE
docs/aem

### DISCUSSION THREAD

**Discussion by giscus** [BriightSpark/aem-community-cms#23] [General]
# docs/aem

Adobe Experience Manager functions as a Content Management System (CMS), a Digital Asset Management (DAM) system, a Cloud Service, and much more with extensive integrations across the Adobe Experience Cloud. Adobe Experience Manager (AEM) is the solution to content complexity. As companies increase in size, they must control and optimise digital media across numerous sites and channels.

http://localhost:3000/docs/aem#giscus

**Comment by gileslauw** [comment]
test

**Comment by nguyentthai96** [comment]
test2
